include/openenclave/internal/plugin.h

```diff
@@ -104,15 +105,17 @@ struct _oe_attester
      * evidence buffer in bytes.
      * @param[out] endorsements_buffer An output pointer that will be assigned
      * the address of the endorsements buffer.
+     * oe_get_evidence() (that calls this entry point) does not perform further
```

Comment:
@shnwc This is the remaining outdated comment that still needs to be removed.

Reply:
Thank you!